tests/StringyTest.php

```diff
@@ -546,6 +546,19 @@ public function testTruncate($expected, $str, $length, $substring = '',
     }
 
     /**
+     * @dataProvider safeTruncateProvider()
+     */
+    public function testSafeTruncate($expected, $str, $length, $substring = '',
+                                     $encoding = null)
```

Comment:
How about merging this PR and, if you wish, opening a separate issue for further discussion concerning CS?